<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="widget_groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>350</height>
      </size>
     </property>
     <property name="title">
      <string>ATTACH</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QFrame" name="widget_GroupFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTableWidget" name="drivers_tableWidget">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>150</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>370</width>
               <height>150</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>150</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>20</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>28</number>
             </attribute>
             <row>
              <property name="text">
               <string>0</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>1</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>2</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>3</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>4</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>5</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>6</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>7</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>8</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>9</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Drivers</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Labels</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QCheckBox" name="constraintSwitch_checkBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>10</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Switch</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QPushButton" name="removeDrivers_pushButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QPushButton" name="addDrivers_pushButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QCheckBox" name="point_checkBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>10</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Point</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QCheckBox" name="orient_checkBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>10</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Orient</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QCheckBox" name="scale_checkBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>10</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Scale</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QCheckBox" name="multiAttrSwitch_checkBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>10</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>MultiAttr</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QGroupBox" name="attachPoint_groupBox">
               <property name="title">
                <string>Attach Point Bool</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_14">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QRadioButton" name="attachPoint_True">
                  <property name="text">
                   <string>True</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QRadioButton" name="attachPoint_False">
                  <property name="text">
                   <string>False</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="attachPointNode_groupBox">
               <property name="title">
                <string>Attach Point Node</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="loadAttachPoint_lineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="loadAttachPoint_pushButton">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="attachOrient_groupBox">
               <property name="title">
                <string>Attach Orient</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_15">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QRadioButton" name="attachOrient_False">
                  <property name="text">
                   <string>False</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QRadioButton" name="attachOrient_True">
                  <property name="text">
                   <string>True</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="attachOrientNode_groupBox">
               <property name="title">
                <string>Attach Orient Node</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QPushButton" name="loadAttachOrient_pushButton">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="loadAttachOrient_lineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="attachScale_groupBox">
               <property name="title">
                <string>Attach Scale</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_18">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QRadioButton" name="attachScale_False">
                  <property name="text">
                   <string>False</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QRadioButton" name="attachScale_True">
                  <property name="text">
                   <string>True</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="attachScaleNode_groupBox">
               <property name="title">
                <string>Attach Scale Node</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="spacing">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QPushButton" name="loadAttachScale_pushButton">
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="loadAttachScale_lineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="config_label_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Config</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="configNode_lineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="loadConfig_pushButton">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Load</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="attrPrefix_label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Attr Prefix</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="attrPrefix_lineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="driven_label">
          <property name="text">
           <string>Driven</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="driven_lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="driven_pushButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
